src/test.h

```diff
@@ -10,15 +15,20 @@
 
 #define UNIQUE_NAME(base) PP_CAT(PP_CAT(base, __LINE__), __COUNTER__)
 
-#define TEST_CASE(name, block) int UNIQUE_NAME(_test_id_) = reg_test((name), [](GameState *game) -> bool block);
+#define _FILE() PP_CAT_II(__FILE__)
+#define _LINE() PP_CAT_II(__LINE__)
```

Comment:
You don't need these since you aren't concatenating anything. So just replace _FILE() with __FILE__ and _LINE() with __LINE__. Macros aren't that wierd. :P

Reply:
You never know, right?